flink-container/kubernetes/README.md

```diff
@@ -42,6 +41,7 @@ At last, you should start the task manager deployment:
 
 You can provide the following additional command line arguments to the cluster entrypoint:
 
+- `--job-classname <job class name>`: Class name of the job to run. By default, the Flink class path is scanned for a JAR with a `Main-Class` or `program-class` manifest entry and chose as the job class. Use this command line argument in order to manually set the job class. This argument is required in case that no or more than one JAR with such a manifest entry is available on the class path.
```

Comment:
chose -> chosen
command line -> command-line
Use argument in order to -> Use argument o

Reply:
chose -> chosen ✅ 
command line -> command-line ❌ 
Use argument in order to -> Use argument o ❓

Reply:
The `in order` isn't necessary.